tests/core/activity/registry_test.py

```diff
@@ -28,6 +31,11 @@ def fixture_form():
     return MagicMock(FormReplyActivity)
 
 
+@pytest.fixture(name="user")
```

Comment:
`userJoined` would be more explanatory IMHO

Reply:
+1, maybe `user_joined` instead of camel case